src/views/Farms/components/Table/Links.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from 'styled-components'
+import useI18n from 'hooks/useI18n'
+import { LinkExternal } from '@pancakeswap-libs/uikit'
+
+export interface LinksProps {
+  bsc: string
+  info: string
+}
+
+const StyledLinkExternal = styled(LinkExternal)`
```

Comment:
if you don't want to display the svg, use the `Link` component , with the `external` props